src/Pulse.php

```diff
@@ -301,7 +301,9 @@ public function store(): int
             $this->entries->filter($this->shouldRecord(...)),
         ));
 
-        Lottery::odds(...$this->app->make('config')->get('pulse.ingest.trim_lottery'))
+        $odds = $this->app->make('config')->get('pulse.ingest.trim.lottery') ?? $this->app->make('config')->get('pulse.ingest.trim_lottery');
+
+        Lottery::odds(...$odds)
```

Comment:
Maintaining backwards compatibility for those apps already out there.

We can ditch this when we tag v1. Added to upgrade guide.